cart/cart.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="cart.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Tenor+Sans&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <button class="btn-style">
+          <img
+            src="../assets/general-assets/close.svg"
+            alt="close"
+            width="35px"
+          />
+        </button>
+        <h2>CART</h2>
+      </nav>
+    
+      <main>
+        <section>
+          <div class="img-container">
+            <img src="../assets/cart-page/cart1.png" alt="NOTFOUND" />
+          </div>
+          <div class="product-info">
+            <h3>LAMEREI</h3>
+            <p>Recycle Bouncle Knit Cardigan Pink</p>
+            <div class="quantity">
+              <button class="btn-style">
+                <img src="../assets/general-assets/minus.svg" alt="close" />
+              </button>
+              <p class="counter">1</p>
+              <button class="btn-style">
+                <img src="../assets/general-assets/Plus.svg" alt="close" />
+              </button>
+            </div>
+            <p class="pricing">$120</p>
+          </div>
+        </section>
+
+        <section>
+            <div class="img-container">
+              <img src="../assets/cart-page/cart2.png" alt="NOTFOUND" />
+            </div>
+            <div class="product-info">
+              <h3>5252 BY O!OI</h3>
+              <p>2021 Signature Sweatshirt [NAVY]</p>
+              <div class="quantity">
+                <button class="btn-style">
+                  <img src="../assets/general-assets/minus.svg" alt="close" />
+                </button>
+                <p class="counter">1</p>
+                <button class="btn-style">
+                  <img src="../assets/general-assets/Plus.svg" alt="close" />
+                </button>
+              </div>
+              <p class="pricing">$120</p>
+            </div>
+          </section>
+      </main>
+    </header>
+    <footer>
+        <div class="sub-total">
+          </span><h3>SUB TOTAL</h3>
+          <h3 class="pricing" style="font-size:1.5rem;">240$</h3>
+        </div>
+        <p class="note">*shipping charges, taxes and discount codes are calculated at the time of accounting.</p>
+    </footer>
+    
+      <button class="buy-btn"><img src="../assets/general-assets/shopping-bag.svg" alt="" > BUY NOW  </button>
+    
```

Comment:
this is the footer of this page, wrap it with a footer tag 
```suggestion
<footer>
      <button class="buy-btn"><img src="../assets/general-assets/shopping-bag.svg" alt="" > BUY NOW  </button>
       </footer>
```

Reply:
also, reduce the size of the footer on big screens

<img width="1438" alt="Screen Shot 2022-02-08 at 3 49 45 PM" src="https://user-images.githubusercontent.com/47992412/153000168-8311988e-5f92-42ef-9dbc-0c2e279318d9.png">